net-mgmt/configsync/src/opnsense/mvc/app/views/VerbNetworks/ConfigSync/settings.volt

```diff
@@ -0,0 +1,206 @@
+                    alert-messages are purely a cosmetic user-interface thing and previous
+                    OPNsense versions are known to work under the hood even if the user
+                    message appears to get "stuck" and not progress.
+                </p>
+                
+                <h2>Example AWS IAM policy</h1>
+                <p>Consider the AWS IAM policy below to restrict the resources and actions available 
+                    to the AWS access-key used.  Of particular note is that this policy does not 
+                    require <code>GetObject</code> actions which hence prevents these credentials
+                    being re-purposed to gain access to previous system configurations via AWS-S3
+                    storage.
+                </p>
+<pre>{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [ "s3:ListBucket" ],
+            "Resource": [ "arn:aws:s3:::BUCKET_NAME" ]
+        },
+        {
+            "Effect": "Allow",
+            "Action": [ "s3:ListBucket", "s3:PutObject" ],
+            "Resource": [ "arn:aws:s3:::BUCKET_NAME/PATH_NAME/*" ]
+        }
+    ]
+}</pre>
+                <p><strong>Note:</strong> Be sure to replace <code>BUCKET_NAME</code> and <code>PATH_NAME</code> with your own values above.</p>
+
+                <hr />
+                
+                <h1>Author</h1>
+                <p>Configuration Sync is a Verb Networks plugin for OPNsense - we make other tools for OPNsense too!</p>
+
+                <h1>License</h1>
+                <p>BSD-2-Clause - see LICENSE file for full details.</p>
+
+                <h1>Copyright</h1>
+                <p>Copyright 2018 - All rights reserved - <a href="https://verbnetworks.com/">Verb Networks Pty Ltd</a></p>
+                
+            </div>
+
+        </div>
+    </div>
+</div>
+
+<style>
+    #configsync\.settings\.StorageFullURI, #configsync\.settings\.SystemHostid {
+        line-height: 34px;
+        display: inline-block;
+        vertical-align: middle;
+        
+        font-size: 80%;
+        font-family: Menlo, Monaco, Consolas, "Courier New", monospace;
+    }
+</style>
+
+<script>
+    
+    /**
+     * updateFullURI
+     */
+    function updateFullURI() {
+        if($('#configsync\\.settings\\.Provider').val() === 'awss3') {
+            var link_url = 'https://s3.console.aws.amazon.com/s3/buckets/';
+            link_url += $('#configsync\\.settings\\.StorageBucket').val() + '/';
+            link_url += $('#configsync\\.settings\\.StoragePath').val() + '/';
+
+            var link_name = 's3://';
+            link_name += $('#configsync\\.settings\\.StorageBucket').val() + '/';
+            link_name += $('#configsync\\.settings\\.StoragePath').val();
+        }
+        $('#configsync\\.settings\\.StorageProviderLink').html('<a target="_blank" href="' + link_url +'">' + link_name + '</a>');
```

Comment:
very likely XSS

Reply:
Not "very likely" it is confirmed, see attached - has been resolved
![screenshot from 2018-08-14 20-08-46](https://user-images.githubusercontent.com/210474/44092608-7ecaf7a4-a003-11e8-9d34-ad8f9615dad3.png)